securedrop/tests/test_journalist.py

```diff
@@ -1145,8 +1146,12 @@ def test_admin_add_user_integrity_error(journalist_app, test_admin, mocker):
                 "error")
 
     log_event = mocked_error_logger.call_args[0][0]
-    assert ("Adding user 'username' failed: (builtins.NoneType) "
-            "None [SQL: 'STATEMENT'] [parameters: 'PARAMETERS']") in log_event
+    if six.PY2:
```

Comment:
Nit: We could extract the `builtin` to a var and format these strings. On first glace I missed the dunder around `builtin` and didn't even see why we were branching.

Reply:
I find this is easier for now as we will throw out all Python2 branch/code in next release.